requirements/core.txt

```diff
@@ -5,8 +5,8 @@
 
 cartopy
 #conda: proj4<5
-cf-units>=2
-cftime==1.0.1
+cf_units>=2
```

Comment:
Can you undo this? 
It would be better to keep the hyphened name

Reply:
I think this came from it still being `cf_units` on `master`, as the fix has gone onto the `v2.2x` branch which has not yet been mergedback onto `master`

Reply:
My mistake, I think I was reading the diffs the wrong way round !